crates/proto/Cargo.toml

```diff
@@ -25,47 +25,63 @@ categories.workspace = true
 license.workspace = true
 
 [features]
-std = ["critical-section/std", "rand/thread_rng"]
-
-async-std = ["dep:async-std", "dep:pin-utils", "dep:socket2"]
-
-tls-aws-lc-rs = ["tokio-rustls/aws-lc-rs", "__tls"]
-https-aws-lc-rs = ["tls-aws-lc-rs", "__https"]
+std = [
+    "critical-section/std",
+    "data-encoding/std",
+    "futures-channel/std",
+    "futures-io/std",
+    "futures-util/std",
+    "ipnet/std",
+    "rand/std",
+    "rand/thread_rng",
+    "ring/std",
+    "thiserror/std",
+    "tracing-subscriber/env-filter",
+    "tracing-subscriber/fmt",
+    "tracing-subscriber/std",
+    "tracing/std",
+    "url/std",
+]
+
+async-std = ["dep:async-std", "dep:pin-utils", "dep:socket2", "std"]
```

Comment:
Note, this is going away in #2867.

Reply:
We'll have to wait for this to merge then

Reply:
Merged.

Reply:
Rebased on main